packages/unenv-preset/src/preset.ts

```diff
@@ -19,6 +19,7 @@ const nativeModules = [
 	"_tls_wrap",
 	"assert",
 	"assert/strict",
+	"async_hooks",
```

Comment:
Does workerd implement the full `async_hooks` package? [The documentation](https://developers.cloudflare.com/workers/runtime-apis/nodejs/asynclocalstorage/) states:

> Workers does not implement the full [async_hooks ↗](https://nodejs.org/dist/latest-v18.x/docs/api/async_hooks.html) API upon which Node.js' implementation of AsyncLocalStorage is built.

Reply:
It implements all the methods that we were polyfilling.